src/coreclr/debug/ee/debugger.cpp

```diff
@@ -16787,6 +16788,13 @@ BOOL Debugger::IsOutOfProcessSetContextEnabled()
 }
 #endif // OUT_OF_PROCESS_SETTHREADCONTEXT
 #endif // DACCESS_COMPILE
+#ifndef DACCESS_COMPILE
+BOOL Debugger::MulticastTraceNextStep(BYTE* pbDel, INT32 count)
+{
+    DebuggerController::DispatchMulticastDelegate(pbDel, count);
+    return TRUE;
+}
```

Comment:
```suggestion
void Debugger::MulticastTraceNextStep(BYTE* pbDel, INT32 count)
{
    DebuggerController::DispatchMulticastDelegate(pbDel, count);
}
```
The return value is always true and never checked. Return `void` instead?